e/error-prone-annotations/build_info.json

```diff
@@ -0,0 +1,15 @@
+{
+    "maintainer": "pratibh-ibm",
+    "package_name": "error-prone",
```

Comment:
Can update package name to error-prone-annotations to match script name, header, folder etc.